web/views/email_view.ex

```diff
@@ -2,23 +2,17 @@ defmodule Cog.EmailView do
   use Cog.Web, :view
   require Logger
 
-  def reset_url(token) do
-    base_url = case Application.get_env(:cog, :password_reset_base_url, nil) do
-      nil ->
-        Logger.warn("""
-        Base URL not set for password reset, defaulting to 'localhost'.
-        The base url is used to generate a password reset url when a request
-        to reset a password is received. The password reset url is generated
-        by appending a token to the base url.
-
-        Please set the env var 'COG_PASSWORD_RESET_BASE_URL'.
-        """)
-
-        %URI{host: "localhost"}
-
-      base_url ->
-        URI.parse(base_url)
+  def webui? do
```

Comment:
I'd also add some documentation to this function... it's not immediately clear why it's named `webuil?`